ucb_prebuild.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This is here to get CloudBuild to play along. We need to make it think that this is a Unity project, so it moves all of the files into the empty Assets folder. Once we get the process working (we need them to add the EVR preview editor build) we will try to refine it so that we can keep the repo clean.